tools/sense_studio/templates/project_details.html

```diff
@@ -12,10 +12,18 @@
         <h3 class="ui header center aligned">Settings</h3>
         <div class="ui toggle checkbox hashoverpopup"
            data-content="Should GPU be used for computing features?"
-           onclick="toggleGPU('{{ path }}');">
+           onclick="toggleGPU('{{ path }}', 'use_gpu');">
```

Comment:
Her in my view, we can make a constant variable for `'use_gpu'` and `'project_temporal'` strings.